tests/Fixer/FunctionNotation/PhpdocToParamTypeFixerTest.php

```diff
@@ -537,6 +537,102 @@ function bar($x) {}
                 fn($a, $b) => 1,
             );',
         ];
+
+        yield 'types defined with "phpstan-type"' => [
+            <<<'PHP'
+                <?php
+                /**
+                 * @phpstan-type _Pair array{int, int}
+                 */
+                class Foo {
+                    /**
+                     * @param _Pair $x
+                     */
+                    public function f($x): void {}
+                }
+                /**
+                 * @phpstan-type _Trio array{int, int, int}
+                 */
+                class Bar {
+                    /**
+                     * @param _Trio $x
+                     */
+                    public function f($x): void {}
+                }
+                PHP,
+        ];
+
+        yield 'types defined with "phpstan-import-type"' => [
+            <<<'PHP'
+                <?php
+                /**
+                 * @phpstan-import-type _Pair from FooFoo
```

Comment:
The import tags can change the type name (alias):

```
@phpstan-import-type _Pair from FooFoo as TPair
```

`TPair` should then be excluded instead of `_Pair`.
But not sure if it is easy to handle with current DocBlock parser?